CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ For details about compatibility between different releases, see the **Commitment
 ## [Unreleased]
 
 ### Added
+- Allow setting an expiry date for API keys in the Console
```

Comment:
```suggestion

- Allow setting an expiry date for API keys in the Console
```

Please internalize the correct whitespacing. All headlines are followed by a whitespace, followed by the list, followed by another whitespace.